internal/core/app_deploy.go

```diff
@@ -144,11 +144,15 @@ type deployOperation struct {
 	// id is populated with the deployment id on Upsert
 	id string
 
+	// sequence is the monotonically incrementing version number of
+	// the deployment
+	sequence uint64
+
 	// cebToken is the token to set for the deployment to auth
 	cebToken string
 
 	// result is either a component.Deployment or component.DeploymentWithUrl
-	result interface{}
+	result   interface{}
```

Comment:
- Can you run `make format`? That should fix the linting test.

Reply:
Done.